opensuse-tomcat-image/src/main/docker/Dockerfile

```diff
@@ -72,9 +57,7 @@ ENV CATALINA_HOME /usr/share/tomcat
 ENV PATH $CATALINA_HOME/bin:$PATH
 WORKDIR $CATALINA_HOME
 
```

Comment:
Now that you are using a staged build in this file with a pre work area the final docker image now does need healthcheck and exposed ports in it. Previously the way you had it with no staged build and one 'FROM' these were not required.
This should have been caught when you compared the logs from this image with the logs from the Master image, you can see a line like this is missing from the logs
[2022-12-08 13:31:30.687Z #30d.027 WARN  -            -   ] AccessLog: 127.0.0.1 - "GET /healthcheck HTTP/1.1" 404 682 "-" "curl/7.79.1"

Reply:
Have now updated locally, can now see the healthcheck logs as expected, will now rebuild on jenkins and await before re-requesting a review 
![image](https://user-images.githubusercontent.com/96247486/206473725-732560df-2bf7-4c53-a6fb-a2eea5058edf.png)

Reply:
Just realised I had a line highlighted in that screenshot which isn't the logs, please ignore, the logs can be seen at the bottom